cpp/include/cugraph/sampling_functions.hpp

```diff
@@ -129,7 +131,7 @@ uniform_neighbor_sample(
   std::optional<raft::device_span<label_t const>> starting_vertex_labels,
   std::optional<std::tuple<raft::device_span<label_t const>, raft::device_span<int32_t const>>>
     label_to_output_comm_rank,
-  raft::host_span<int32_t const> fan_out,
+  std::optional<raft::host_span<int32_t const>> fan_out,
```

Comment:
Why are we updating a deprecated method?

Reply:
Right. I didn't revert the changes. Doing it now